langlib/lang.error/src/main/ballerina/error.bal

```diff
@@ -51,6 +63,26 @@ public isolated function cause(error e) returns error? = @java:Method {
 
 # Returns the error's detail record.
 #
+# ```ballerina
+# string file = "test.bal";
+#
```

Comment:
We need to remove L68 ryt?
It should be consistent with L51-54

Reply:
Added a new line because `file` is used in both `err` and `fileNotFoundErr`.